packages/components/atoms/f-button/package.json

```diff
@@ -31,7 +31,7 @@
     "lint": "vue-cli-service lint",
     "lint:fix": "yarn lint --fix",
     "lint:style": "vue-cli-service lint:style",
-    "test": "vue-cli-service test:unit",
+    "test": "vue-cli-service test:unit && cd ../../../../ && yarn allure:clean",
```

Comment:
Looks like you're missing the `allure:clean` task in some of the packages?

Reply:
thank you! accidentally deleted some 🤦